examples/complete/main.tf

```diff
@@ -37,3 +37,32 @@ module "create_scc_instance" {
   en_instance_crn                   = module.event_notification.crn
   skip_cos_iam_authorization_policy = false
 }
+
+locals {
+  scope = {
+    environment = var.scc_scope_environment
+    properties = [
+      {
+        name  = "scope_id"
+        value = module.resource_group.resource_group_id
+      },
+      {
+        name  = "scope_type"
+        value = "account.resource_group"
+      }
+    ]
+  }
+}
+
+module "create_profile_attachment" {
+  count                  = var.scc_profile_id == null ? 0 : 1
+  source                 = "../../modules/attachments/"
+  profile_id             = var.scc_profile_id
```

Comment:
Find out the profile ID of one of the predefined profiles, and hard code it here until provider add support to do data lookup of profile using a name